test/tasks/common_test.go

```diff
@@ -84,6 +85,7 @@ func runTaskTestCases(t *testing.T, taskName string, testCases map[string]taskte
 				Timeout:                 5 * time.Minute, // depending on  the task we may need to increase or decrease it
 				AlwaysKeepTmpWorkspaces: *alwaysKeepTmpWorkspacesFlag,
 			})
+			t.Logf("Test execution time: %fs", time.Since(start).Seconds())
```

Comment:
@michaelsauter Would it make sense to include the test name as part of the log msg? This way, we can tell in GitHub Actions what log message corresponds to what test case

Reply:
Right, we could add this. For now I just wanted to have a quick way to be able to track where stuff is taking long. But adding the name in wouldn't hurt ...